### DISCUSSION TITLE
Disable autorun at boot

### DISCUSSION THREAD

**Discussion by raako71** [iterate-ch/cyberduck#15997] [Q&A] [answered]
feature request for a checkbox in config to disable auto starting the app at user login / boot (for mountain duck)

**Comment by AliveDevil** [comment, marked as answer]
Already implemented: [Enable login item](https://docs.mountainduck.io/mountainduck/preferences/#login-item).

**Reply by raako71** [reply]
Thank you. Could I suggest a rename: 
"Autostart app"
"Autostart app on user login"

**Reply by AliveDevil** [reply]
Well, we don't have separate localizations for macOS/Windows, so "Login item" comes from macOS, where the autostart entries are called login items.

@dkocher Any chance we can add Windows-localized string for autostart?

**Reply by raako71** [reply]
ahh cool, all good :)
